src/cloudant/_common_util.py

```diff
@@ -160,7 +162,7 @@ def feed_arg_types(feed_type):
         return _COUCH_DB_UPDATES_ARG_TYPES
     return _CHANGES_ARG_TYPES
 
-def python_to_couch(options):
+def python_to_couch(options, encoder=None):
     """
     Translates query options from python style options into CouchDB/Cloudant
```

Comment:
I think we'll want to add a comment about the new `encoder` parameter.

Reply:
Thanks! I moved it to the database tests in f5610d5.